Automattic/wp-calypso
# CPT/State: Add state actions and reducer for managing post type taxonomies

This pull request seeks to add state handlers necessary to manage post type taxonomies. This will be used as part of the current effort to enable custom post type editing as a prerequisite for post-type-agnostic terms assignment.

__Implementation notes:__

This adds a new `postTypeTaxonomies` undocumented wpcom.js method. In parallel, I've submitted a pull request for this endpoint to be added to the base wpcom.js library in Automattic/wpcom.js#193

__Testing instructions:__

The included state is not yet used anywhere in the application, but verify that its inclusion does not introduce any regressions while loading and navigating Calypso.

Ensure that included Mocha tests pass:

```bash
npm test
```
